Cargo.lock

```diff
@@ -48,7 +48,7 @@ dependencies = [
 
 [[package]]
 name = "rust-sudoku"
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
Thanks, I guess I didn't rebuild the project after bumping the `Cargo.toml` version number.